src/main/java/com/CezaryZal/authentication/Bcrypt.java

```diff
@@ -0,0 +1,15 @@
+package com.CezaryZal.authentication;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+@Component
+public class Bcrypt {
+
+    @Bean
+    PasswordEncoder passwordEncoder(){
+        return new BCryptPasswordEncoder();
```

Comment:
To możesz sobie wrzucić w SecurityConfig, wtedy nie potrzebujesz tej klasy.

Reply:
Ok, przeniosłem.
Myślałem żeby zrobić oddzielną klasę na tego beana ponieważ korzystam z niego nie tylko w klasie SecurotyConfig.